src/crypto/tls/conn.go

```diff
@@ -50,6 +50,9 @@ type Conn struct {
 	// verifiedChains contains the certificate chains that we built, as
 	// opposed to the ones presented by the server.
 	verifiedChains [][]*x509.Certificate
+	// verifiedDC is contains a delegated credential from the peer and
+	// has been verified against the leaf certificate.
```

Comment:
at least, when it was provided and valid.

Reply:
Added as well.